recipes/parallelproj/meta.yaml

```
@@ -0,0 +1,44 @@
+{% set name = "parallelproj" %}
+{% set version = "1.2.6" %}
+{% set build_number = "0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/gschramm/parallelproj/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: 826a83e51db6459004d17c030c21fb32e7cb5205a45c28c8f50f87f235321196
+
+build:
+  number: {{ build_number }}
+  string: "h{{ PKG_HASH }}_{{ build_number }}_cpu"   # [cuda_compiler_version == "None"]
+  string: "h{{ PKG_HASH }}_{{ build_number }}_cuda"  # [cuda_compiler_version != "None"]
```

```suggestion
  string: "cpuh{{ PKG_HASH }}_{{ build_number }}"   # [cuda_compiler_version == "None"]
  string: "cuda{{ cuda_compiler_version|string }}h{{ PKG_HASH }}_{{ build_number }}"  # [cuda_compiler_version != "None"]
```
Convention is to prefix the build string with `cuda + the toolkit version`. The build number should be the last thing in the string. You may use `{{ cuda_compiler_version | replace('.', '') }}` if you don't want the period.